### PR TITLE
Update meta.yaml to hard copy the Parameters directory

### PR DESCRIPTION
**Pull request by TomSlater**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

This PR will hopefully add the Parameters directory to the built package.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -30,6 +30,11 @@ test:
   imports:
     - ParticleSpy
 
+outputs:
```
Comment (tschoonj) [inline recipe/meta.yaml:33]: This won't do what you are hoping for.

Instead this should be done by placing an empty `__init__.py` file in the Parameters subfolder, as well as appropriate modifications to `MANIFEST.in`, which will ensure this folder gets included in the source tarball and wheel.

**Review by tschoonj — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (TomSlater) [reply recipe/meta.yaml:33]: Thanks for the help. I'll try this instead.

Comment (tschoonj) [reply recipe/meta.yaml:33]: Your MANIFEST.in looks actually good, but I think that the Parameters don't get included as you are not including the `ParticleSpy` toplevel folder. 

You can also try something like:

```
recursive-include ParticleSpy/Parameters *
recursive-include ParticleSpy/Data *
```

Comment (TomSlater) [reply recipe/meta.yaml:33]: Is this done differently via pip and conda? The directories are included when using pip but not when using conda.

Comment (tschoonj) [reply recipe/meta.yaml:33]: There is indeed a difference: with pip you will install the wheel, which has those files included, but conda installs using the source tarball, which doesn't have them.

**Pull request closed without merging**